scheduler/tests/util.py

```diff
@@ -1,10 +1,104 @@
 """
 Test utilities
 """
+from scheduler.classes.Member import Member
+from scheduler.classes.Driver import Driver
+from scheduler.classes.Rider import Rider
+import scheduler.classes.Day as Day
 
 
-def sort_by_name(dct):
+def convert_days(days: list) -> list:
+    """Converts a list of dictionary days to DayInfo objects
+
+    This is a TODO item. This function works, but it was written to get tests working again.
+    It might not be needed once a better testing framework gets worked out.
+
+
+        Args:
+            days:
+                a list of dictionaries containing the required information to create a DayInfo object
+        
+        Returns:
+            A DayInfo object
+    """
+
+    dayInfos = list()
+
+    for d in days:
+        day = Day.DayInfo(day=Day.from_str(get_with_check(d, "day")),
+                          times=get_with_check(d, "departure_times"),
+                          locations=get_with_check(d, "locations"))
+
+        dayInfos.append(day)
+
+    return dayInfos
+
+
+def get_with_check(d: dict, key: str):
```

Comment:
what does this do? I see it checks first if the key is in the dictionary, but it seems weird to return a list() if it isn't. This might be a better place for an exception?

Regardless, a comment should be added to clarify the intent. The returning of a d[key] or list() makes the intent non trivial

Reply:
Returning d[key] or None instead. See most recent commit for more details.